medusa/failed_processor.py

```diff
@@ -37,11 +29,11 @@ def __init__(self, dirName, nzbName):
 
     def process(self):
         """
-        Do the actual work
+        Do the actual work.
 
         :return: True
         """
-        self._log(u'Failed download detected: ({nzb}, {dir})'.format(nzb=self.nzb_name, dir=self.dir_name))
+        self._log(u'Failed download detected: ({nzb}, {dir})', {'nzb': self.nzb_name, 'dir': self.dir_name})
```

Comment:
A simple log adapter could work, however I don't think one is necessary since it would see such limited use.

In my opinion `self._log` should be renamed `self.log` since it's the method we call directly.  The args/kwargs for `self.log` should get fixed so they match that of `logger.log`.

The `BraceAdapter` should not be used as `self.log` modifies everything regardless of level, so lazy formatting is not necessary.